content/blog/2017/08/2017-08-11-remoting-update.adoc

```diff
@@ -19,7 +19,7 @@ agents, please read this announcement carefully.
 If you have ever seen messages like "Channel is already closed" or "Remote call failed" in your build logs,
 you have already met link:/projects/remoting[Jenkins Remoting].
 
-Remoting is an agent executable (aka `slave.jar`) and a library implementing the communication layer between Jenkins masters and their agents (including communication protocols, distributed calls and classloading).
+Remoting is an agent executable and a library implementing the communication layer between Jenkins masters and their agents (including communication protocols, distributed calls and classloading).
```

Comment:
There is an agent.jar that replaces the former named item, but we should complete remove the old jar.

Reply:
Okay, corrected